command/agent/periodic_endpoint.go

```diff
@@ -0,0 +1,40 @@
+package agent
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/nomad/nomad/structs"
+)
+
+func (s *HTTPServer) PeriodicSpecificReqeust(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	path := strings.TrimPrefix(req.URL.Path, "/v1/periodic/")
+	switch {
+	case strings.HasSuffix(path, "/force"):
+		jobName := strings.TrimSuffix(path, "/force")
+		return s.periodicForceRequest(resp, req, jobName)
+	default:
+		return nil, CodedError(405, ErrInvalidMethod)
```

Comment:
Should be `404` since we want to respond the the caller saying the URL isn't valid. 405 should be returned in the beginning if the method is anything else other than `PUT`. I think `POST` is more appropriate but looks like our client uses `PUT` instead of `POST`